test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTests.cs

```diff
@@ -90,16 +99,27 @@ public void TraceExportResultIsSuccess(OtlpExportProtocol protocol, string endpo
             activity?.Stop();
 
             Assert.NotNull(delegatingExporter);
-            Assert.True(tracerProvider.ForceFlush());
+
+            if (forceFlush)
+            {
+                Assert.True(tracerProvider.ForceFlush());
+            }
+            else if (exporterOptions.ExportProcessorType == ExportProcessorType.Batch)
```

Comment:
Do you plan to test `ExportProcessorType.Simple` as well?

Reply:
Originally I had planned to just cover the default options but it does seem good to have this. Added.